runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java

```diff
@@ -161,6 +202,68 @@ public void onWallObjectSpawned(WallObjectSpawned event)
 		}
 	}
 
+	@Subscribe
+	public void onMenuOptionClicked(MenuOptionClicked event)
+	{
+		if (event.getWidgetId() != WidgetInfo.INVENTORY.getId())
```

Comment:
This needs a check for the `RUNELITE` Menu Action type, so that it won't trigger while specifying Shift Click options
```suggestion
		if (event.getWidgetId() != WidgetInfo.INVENTORY.getId() || event.getMenuAction() == MenuAction.RUNELITE)
```